### ISSUE TITLE
[Issue] Conflict Between maz-ui and valibot Dependencies

### ISSUE THREAD

**Issue by devhardiyanto** [LouisMazel/maz-ui#1134]
### Issue Summary

**Description**:
Encountered a peer dependency conflict while trying to install `maz-ui@3.45.1` with `valibot@0.37.0` in the project. The `maz-ui` package has a peer dependency on `valibot` version `^0.30.0`, which conflicts with the installed version `0.37.0`.

**Error Message**:
```
npm ERR! Found: valibot@0.37.0
npm ERR! node_modules/valibot
npm ERR!   valibot@"^0.37.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional valibot@"^0.30.0" from maz-ui@3.45.1
npm ERR! node_modules/maz-ui
npm ERR!   maz-ui@"^3.45.1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: valibot@0.30.0
npm ERR! node_modules/valibot
npm ERR!   peerOptional valibot@"^0.30.0" from maz-ui@3.45.1
npm ERR!   node_modules/maz-ui
npm ERR!     maz-ui@"^3.45.1" from the root project
```

### Steps to Reproduce
2. Install `maz-ui` as a dependency.
3. Attempt to install `valibot@0.37.0`.
4. Run `npm install`.

### Expected Behavior
The installation should succeed without any peer dependency conflicts.

### Actual Behavior
npm throws an error due to a conflicting peer dependency between `maz-ui@3.45.1` and `valibot@0.37.0`. `maz-ui` requires `valibot` version `^0.30.0`, but the project is using `valibot@0.37.0`, leading to a conflict.

### Suggested Resolution
Consider updating the `maz-ui` package to support the latest version of `valibot`, or relax the peer dependency requirement to allow compatibility with `valibot@0.37.0`. Alternatively, if there's a specific reason `maz-ui` depends on `valibot@^0.30.0`, updating the documentation to clarify the dependency requirements would be helpful.

### Environment
- `npm` version: v10.2.3
- `node` version: v20.10.0
- Operating System: Linux Ubuntu-22

Thank you for looking into this issue!

**Comment by devhardiyanto** [comment]
is it okay, if i use `--force` for running the package?

**Comment by LouisMazel** [comment]
Fixed into v3.45.2

Thanks for your feedback!
